blocks/commerce-mini-cart/commerce-mini-cart.js

```diff
@@ -48,6 +61,72 @@ export default async function decorate(block) {
     }, 3000);
   };
 
+  // Handle Edit Button Click
+  async function handleEditButtonClick(cartItem) {
+    try {
+      // Create mini PDP content
+      const miniPDPContent = await createMiniPDP(
+        cartItem,
+        (_updateData) => {
+          const productName = cartItem.name
+            || cartItem.product?.name
+            || placeholders?.Global?.CartUpdatedProductName;
+          const message = placeholders?.Global?.CartUpdatedProductMessage?.replace(
+            '{product}',
+            productName,
+          );
+
+          // Show message in the main cart page
+          const cartNotification = document.querySelector(
+            '.cart__notification',
+          );
+          if (cartNotification) {
+            UI.render(InLineAlert, {
+              heading: message,
+              type: 'success',
+              variant: 'primary',
+              icon: h(Icon, { source: 'CheckWithCircle' }),
+              'aria-live': 'assertive',
+              role: 'alert',
+              onDismiss: () => {
+                cartNotification.innerHTML = '';
+              },
+            })(cartNotification);
+
+            // Auto-dismiss after 5 seconds
+            setTimeout(() => {
+              cartNotification.innerHTML = '';
+            }, 5000);
+          }
+
+          // Also trigger message in the mini-cart
+          showMessage(message);
+        },
+        () => {
+          if (currentModal) {
+            currentModal.removeModal();
+            currentModal = null;
+          }
+        },
+      );
```

Comment:
Like in the previous comment, could you consider removing the element from the DOM instead of clearing its inner HTML to avoid potential virtual DOM side effects?